<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>476</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>645</width>
    <height>476</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>645</width>
    <height>476</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mistery Game</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/icon.jpg</normaloff>../images/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>431</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>641</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../images/intro.qrc">:/newPrefix/first.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Rockwell Extra Bold&quot;;
color: rgb(248, 248, 248);
</string>
    </property>
    <property name="text">
     <string>Misterul de la conacul Wilton</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="custom_text_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 10pt &quot;MS UI Gothic&quot;;
color: rgb(248, 248, 248);</string>
    </property>
    <property name="text">
     <string>Puzzle game cu poveste.</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="start_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Rockwell Extra Bold&quot;;
border: 0px solid black; 
color: rgb(248, 248, 248);

</string>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images/intro.qrc"/>
  <include location="intro.qrc"/>
 </resources>
 <connections/>
</ui>
